model/model_eval/manual/sampling_report.py

```diff
@@ -226,6 +226,7 @@ def parse_args():
     parser.add_argument("--num-samples", type=int, default=2, help="number of sampling runs per configuration")
     parser.add_argument("--config", type=str, default="config/default.json", help="configuration file path")
     parser.add_argument("--half", action="store_true", default=False, help="use float16")
+    parser.add_argument("--quantize", action="store_true", default=False, help="use 8bit quantization")
```

Comment:
Can we call this `--int8`? Could be confusing as-is, as `--half` is also quantisation

Reply:
Sure, I copied the flag name from `inference/worker/worker_full_main.sh` but I can commit a change to `--int8` for this script